python/lsst/cp/pipe/ptc.py

```diff
@@ -446,12 +413,42 @@ def runDataRef(self, dataRef, visitPairs):
 
             butler.put(linearizer, datasetType='Linearizer', dataId={'detector': detNum,
                        'detectorName': detName, 'calibDate': calibDate})
-
-        self.log.info(f"Writing PTC data to {dataRef.getUri(write=True)}")
+        rerunDir = list(dataRef.getButler().storage.repositoryCfgs.keys())[0]
+        self.log.info(f"Writing PTC data to {rerunDir} directory.")
         dataRef.put(datasetPtc, datasetType="photonTransferCurveDataset")
 
         return pipeBase.Struct(exitStatus=0)
 
+    def makePairs(self, dataRefList):
+        """Produce a list of flat pairs indexed by exposure time.
+
+        Parameters
+        ----------
+        dataReflist : `list` [`lsst.daf.peristence.ButlerDataRef`]
+            Data references for exposures for detectors to process.
+
+        Return
+        ------
+        sortedPairs : `dict`
```

Comment:
I think you need to say dict of what? Keys are exposure time in seconds and values are list of `Exposure`? But only the first two exposures that have that same exposure time?

Reply:
Yes, I agree, I'll add more information about the dictionary: the keys and values are as you describe. The input exposures are not necessarily sorted by ascending time in the command line, so this function organizes them in a dictionary. I can change the name sorted to something else (e.g.,  just "pairs").

Reply:
I definitely would not use `sorted` anywhere in the function if nothing is doing any sorting.